include/depthimage_to_laserscan/DepthImageToLaserScan.h

```diff
@@ -206,11 +209,21 @@ namespace depthimage_to_laserscan
       double tf_basis_2_2 = transform.getBasis()[2][2];
       double tf_origin_z = transform.getOrigin().z();
 
-      for(int v = offset; v < offset+scan_height_; v++, depth_row += row_step){
+      // determine lower bound and upper_bound of pixel row to scan
+      // given height_min_, range_min_ and range_max_, calculate the corresponding row in the image and start depth image conversion from there
+      int lower_bound = (height_min_ - tf_origin_z) / range_min_ / 1000 / constant_y + center_y;
```

Comment:
It looks good, lets just go over this line in person to make sure it's 100% correct.